ClinGen/clincoded
# Connects to #927. Indel and haplotype variants.

**Technical details:**
This PR primarily implements the UIs to display the "Data is currently only returned for single nucleotide variants" message in the tables on the "Population" and "Predictors" tabs (rather than trying to display data), when the user-selected variant is not a single nucleotide variant.

**Please use the following IDs while testing different use cases:**
duplication - Variant ID 17677
deletion - Variant ID 55555
deletion/insertion - Variant ID 179355
insertion - Variant ID 37401
inversion - Variant ID 242209
haplotypes - Variant IDs 7901 and 14180

**Steps to test:**
1. Login and use one of the IDs given above at the variation selection interface.
2. Proceed to the **"Population"** tab and expect to see the _"Data is currently only returned for single nucleotide variants. Search *** for this variant."_ message in the 3 tables.
3. Proceed to the **"Predictors"** tab and expect to see the _"Data is currently only returned for single nucleotide variants."_ message in the "Other Predictors" and "Conservation Analysis" tables, and the _"These predictors only return data for missense variants."_ message in the "ClinGen Predictors" table.
